components/rewards/RewardsHome.js

```diff
@@ -25,11 +34,32 @@ function createList(n) {
   return list;
 }
 
-function RewardsHome({ customer }) {
+function RewardsHome({ customer, participating }) {
+  const navigation = useNavigation();
   const rewardsAvailable = parseInt(customer.points, 10) / rewardPointValue;
   const pointsToNext = parseInt(customer.points, 10) % rewardPointValue;
   return (
     <ScrollView style={{ marginLeft: 16, paddingRight: 16 }}>
+      <ColumnContainer>
+        <Overline style={{ marginTop: 24, marginBottom: 4 }}>
```

Comment:
See the latest designs -- this is moved to the bottom (under Avail rewards).

Also, this is included in the rewards screen even in Guest Mode